src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -59,6 +59,8 @@
     protected $startTime;
     protected $loadClassCache;
 
+    private $projectDir;
```

Comment:
@iltar Following your argument, [this line](https://github.com/fabpot/symfony/blob/master/src/Symfony/Component/HttpKernel/Kernel.php#L607) should use `$this->getProjectDir()` then. The current code is inconsistent and I suggest that we change `private $projectDir` to `protected $projectDir` so it matches `protected $rootDir`.